### ISSUE TITLE
Error: License activation strategy could not be determined

### ISSUE THREAD

**Issue by nintendaii** [game-ci/unity-builder#281]
Hi! I am pretty new here. I tried to build my Unity project via unity-builder but got this error. It looks like it needs some docker 
file with env variables BUT I didn’t found anything about docker set up in [builder section.](https://game.ci/docs/github/builder)
https://github.com/game-ci/unity-builder/issues/78#issuecomment-605524307 didn’t helped.
<img width="1346" alt="Screenshot 2021-08-02 at 18 07 45" src="https://user-images.githubusercontent.com/44233090/127883194-dc17f344-22d7-4883-ba3a-6afc20d18a54.png">
Here is my yaml file code (default file from the tutorial except `allowDirtyBuild` flag):
```yaml
name: Build for Android 📱

on: [push, pull_request]

jobs:
  buildForAllSupportedPlatforms:
    name: Build for ${{ matrix.targetPlatform }}
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix:
        targetPlatform:
          - iOS # Build an iOS player.
          - Android # Build an Android .apk standalone app.
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
          lfs: true
      - uses: actions/cache@v2
        with:
          path: Library
          key: Library-${{ matrix.targetPlatform }}
          restore-keys: Library-
      - uses: game-ci/unity-builder@v2
        with:
          allowDirtyBuild: true
      - uses: game-ci/unity-builder@v2
        env:
          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
        with:
          targetPlatform: ${{ matrix.targetPlatform }}
      - uses: actions/upload-artifact@v2
        with:
          name: Build-${{ matrix.targetPlatform }}
          path: build/${{ matrix.targetPlatform }}```

**Comment by webbertakken** [comment]
As the output reads: "License activation strategy could not be determined".

This means you have not set UNITY_LICENSE in your first invocation of unity builder in your case.

We do not recommend using `allowDirtyBuild`, but instead using an up-to-date gitignore file or fixing the workflow that causes new files to be created in your working tree.

**Comment by nintendaii** [comment]
Thanks for the quick reply! It is strange because I have a UNITY_LICENSE variable in my GitHub Secrets. Any advice how to fix this?

**Comment by webbertakken** [comment]
Yep. Read the second line of my first response once more and then look closely at the snippet that you provided in your opening post.

It looks like a copy pasting error.

**Comment by nintendaii** [comment]
Thats great, but I already did the UNITY_LICENSE setting (second workflow from the bottom at the screenshot). I uploaded the artifact to the https://license.unity3d.com/manual. Then I downloaded and pasted the Unity_v20XX.x.ulf file into the Secrets variable. I am trying to understand what I did wrong. Is it possible that I am using wrong license? I copied the code from the manual and got the 2019.2.11f1 license, but in my project I am using Unity version 2021.1.5f. Could this be the root of my problem?
<img width="912" alt="Screenshot 2021-08-02 at 20 02 01" src="https://user-images.githubusercontent.com/44233090/127897912-f5a37809-cc4f-406d-8e63-b5634136293d.png">
